packages/cozy-stack-client/src/logger.js

```diff
@@ -0,0 +1,6 @@
+import minilog from '@cozy/minilog'
```

Comment:
In services/connectors we use cozy-logger: https://github.com/cozy/cozy-libs/tree/master/packages/cozy-logger . Eventually, do we plan to use only one of them to log them all?

Reply:
That's the goal at the end, to have only one logger that handle correctly browser & node. Though,  I don't know the path to do that